report-service/src/main/java/gov/cdc/dataingestion/config/KafkaConfig.java

```diff
@@ -18,13 +18,13 @@
 @EnableKafka
 @Configuration
 public class KafkaConfig {
-    @Value("${spring.kafka.group-id}")
+    @Value("${spring.kafka.group-id:}")
     private String groupId = "";
 
-    @Value("${spring.kafka.bootstrap-servers}")
+    @Value("${spring.kafka.bootstrap-servers:}")
```

Comment:
What are the : at the end of the variable name?

Reply:
That is for default value. Here default value is empty string.